generalized_linear_regression/generalized_linear_regression.md

```diff
@@ -0,0 +1,408 @@
+version:  1.0.0
+current_version_description: Initial version
+module_type: standard
+docs_version: 4.0.0
+language: en
+narrator: UK English Female
+mode: Textbook
+
+title: Generalized Linear Regression
+
+comment:  What is generalized linear regression (including logistic regression) and when might you need it? 
+
+long_description: This lesson walks you through what generalized linear models are, and how they are similar to and different from ordinary linear regression. It uses logistic regression as a specific example, but touches on other generalized linear models as well. This is a theoretical overview, not hands-on coding practice, so no experience with any particular software is expected. If you want to understand generalized linear models better, regardless of what statistical software you use, this is for you!
+
+estimated_time_in_minutes: 60
+
+@pre_reqs
+Learners should already be familiar with the following concepts in statistics and math:
+
+- linear regression (also called "ordinary least squares (OLS) linear regression")
+- the equation of a line (intercept and slope)
+@end
+
+@learning_objectives  
+After completion of this module, learners will be able to:
+
+- describe how generalized linear models are similar to and different from linear models
+- define a "link function"
+- identify when a logistic regression model is appropriate
+@end
+
+good_first_module: false
+data_task: data_analysis
+collection: statistics
+coding_required: false
+
+@sets_you_up_for
+
+@end
+
+@depends_on_knowledge_available_in
+- intro_to_nhst
+@end
+
+@version_history 
+No previous versions.
+@end
+
+import: https://raw.githubusercontent.com/arcus/education_modules/main/_module_templates/macros.md
+-->
+
+# Generalized Linear Regression
+
+@overview
+
+## Linear models: A quick review
+
+Many of the most common statistical tests are applications of one big overarching statistical approach, called the **General Linear Model**. 
+Correlation, t-tests, ANOVAs, and linear regression are all examples of linear models. 
+
+A linear model is anything that can be expressed as an equation for a line: 
+
+$$
+Y = \beta_0 + \beta_1 * X_1 + \beta_2 * X_2 ... \beta_n * X_n + e 
+$$
+
+<div class = "care">
+<b style="color: rgb(var(--color-highlight));">A little encouragement...</b><br>
+
+Even if you're used to running regression models in your own analyses, you may feel like your grasp of the math behind linear models is not strong. 
+You're not alone!
+
+For an approachable review of linear regression models, including a review of the equation behind them, check out this [article explaining linear regression](https://education.arcus.chop.edu/ordinary_linear_regression/).
+
+</div>
+
+For example, we could express a model [predicting fetal weight from sonographic measurements of femur length](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6077071/) as the equation:
+
+$$
+fetal\_weight = \beta_0 + \beta_{femoral_length} * femoral_length + e
```

Comment:
```suggestion
fetal\_weight = \beta_0 + \beta_{femoral\_length} * femoral\_length + e
```

Another general suggestion (coming from my pedantry in the world of mathematics) is that variables like `femoral_weight` should not be spelled out in "math mode" at all, but that might make them look even worse to people who aren't me. But also that has a lot to do with the LaTex interpreter being used and I don't think I can adequately demonstrate here. 

$$
fetal \_weight = \beta_0 + \beta_{femoral \_length} * femoral \_length + e
$$

might be easier to read as

$$
\text{fetal} \textunderscore \text{weight} = \beta_0 + \beta_{\text{femoral} \textunderscore\text{length}} * \text{femoral} \textunderscore\text{length} + e
$$

Github is rendering this a bit whacky but it sorta does that for all long math lines. I'm not going to dig this hole in my brain any deeper unless you want to talk about it :)

Reply:
@rosemm Apparently liascript doesn't like `\textunderscore` 😢